eclair-core/src/main/scala/fr/acinq/eclair/blockchain/bitcoind/ZmqWatcher.scala

```diff
@@ -229,19 +234,32 @@ private class ZmqWatcher(chainHash: ByteVector32, blockCount: AtomicLong, client
           }
         Behaviors.same
 
-      case ProcessNewBlock(block) =>
-        log.debug("received blockid={}", block.blockId)
+      case ProcessNewBlock(blockHash) =>
+        log.debug("received blockhash={}", blockHash)
         log.debug("scheduling a new task to check on tx confirmations")
+        // we have received a block, so we can reset the block timeout timer
+        timers.startSingleTimer(TickBlockTimeout, TickBlockTimeout, blockTimeout)
         // we do this to avoid herd effects in testing when generating a lots of blocks in a row
         timers.startSingleTimer(TickNewBlock, TickNewBlock, 2 seconds)
```

Comment:
```suggestion
        timers.startSingleTimer(TickBlockTimeout, blockTimeout)
        // we do this to avoid herd effects in testing when generating a lots of blocks in a row
        timers.startSingleTimer(TickNewBlock, 2 seconds)
```

Reply:
Done in https://github.com/ACINQ/eclair/pull/1910/commits/46935832f41f5a6b4a1c8ce4d57e7acf0844815d